spec/openapi.yaml

```diff
@@ -1023,25 +837,32 @@ components:
               type: string
             download:
               type: string
+            thumbnailSmall:     # do not send if metadata not ready
```

Comment:
Is it not better only to send the bigger and thumbnail? The page can scale it.

Reply:
Which the url? `/api/thumbnail` ?

Reply:
Filemanager0 will unpack both sizes so why do not send both and pick more suitable image in UI? It will reduce the traffic if you do not need to send big thumbnails for multiple projects (if user is looking on project list)

Reply:
ok